frontend/explorer/public/css/main.css

```diff
@@ -124,3 +124,23 @@ span.search-form-td {
 .hidden {
 	display: none;
 }
+
+#nextbutton {
```

Comment:
why are you adding the CSS of another feature? Please remove and only keep the CSS applied for _this_ feature